packages/oidc-dpop-client-browser/src/dpop/dpop.spec.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { describe, it } from "@jest/globals";
+
+import { generateJWK, generateDpopKey, generateRsaKey } from "./dpop";
+
+describe("generateJWK", () => {
+  it("can generate a RSA-based JWK", async () => {
+    const key = await generateJWK("RSA");
+    expect(key.kty).toEqual("RSA");
+  });
+
+  it("can generate an elliptic curve-based JWK", async () => {
+    const key = await generateJWK("EC", "P-256");
+    expect(key.kty).toEqual("EC");
+  });
+});
+
+describe("generateDpopKey", () => {
+  it("generates an elliptic curve-base key", async () => {
```

Comment:
This seems to be implying that a DPoP key simply **_is_** an elliptic curve-based key, but that doesn't seem right. Question then is, so what is a `DPoP key` exactly (isn't it just a JWK that happens to be generated with certain params)...?

Reply:
Fixed in 28296d3cc4a69bfaf52e4daa6d2f493bd21211b6, and the method name has been changed in cb2893aa360c62d8cb24e2da88f1e033558eb372